astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -252,7 +266,8 @@ def main():
         opts.ignore_keywords = []
         opts.ignore_comments = []
         opts.ignore_fields = []
-        opts.tolerance = 0.0
```

Comment:
No, wait. Now I am confused. In the parser above, there is a `dest="rtol"` for `-d` option. But then, it is no handling for when both `-d` and `-r` are defined. And then below, somehow there is a `tolerance=opts.tolerance` option. I think all these need some cleaning up.